Channels/Integration/01a_Shopware6.md

```diff
@@ -0,0 +1,172 @@
+
+  > [Info] In Shopware 6 there is a standard language, which is configured during the installation and cannot be subsequently edited. The standard language cannot be detected via API and must be therefore manually configured in the *Actindo Core1 Platform* after the connection has been established. When uploading products, Shopware 6 requires some fields to be filled for the standard language.
+
+  ![Edit connection settings](../../Assets/Screenshots/Channels/Settings/Connections/Shopware6/EditConnectionSettings.png "[Edit connection settings]")
+
+3. Click the *Orders* menu entry in the left side bar.  
+  The import settings for orders are displayed on the right side.  
+
+4. Configure the following settings:
+  + If desired, enter an order number in the *Import only orders with a higher number than this* field. Only orders with a higher order number than the specified number will be imported.  
+
+  + Enable the toggles of all statuses for which the orders are to be imported. All statuses available in Shopware 6 are displayed.
+      
+  + If desired, click the *Order state to set in shop after successful import* drop-down list and select the state that will be set for the order if the import has been successful. All states available in Shopware 6 are displayed.
+
+    > [Info] The order states in Shopware 6 have fixed transitions. This means that once an order has a certain state, only specific states can be selected for the next state of the order. Make sure that the selected state is also supported by Shopware 6.    
+
+  + If desired, click the *Order state to set in shop after the order is fully shipped* drop-down list and select the state that will be set for the order if the shipment has been completed. All states available in Shopware 6 are displayed.
+ 
+  + If desired, enable for each payment method the toggles of all states for which the orders are to be imported. If you select no payment states for a certain payment method, all orders with this payment method, regardless their state, will be imported. All states available in Shopware 6 are displayed for each payment method. 
+  The following payment methods are available:   
+    - *Paypal*  
+    - *Cash on delivery*  
+    - *Prepayment*  
+    - *Pay upon invoice*  
+    - *Ratepay Prepayment*
+    - *Ratepay Direct Debit*
+    - *Ratepay Open invoice*
+    - *Ratepay Payment by Installment*
+    - *Ratepay 0% Financing*
+    - *PAYONE Apple Pay*
+    - *Credit card (Visa/Mastercard)*
+    - *Payment SEPA Direct Debit*
+    - *PAYONE PayPal*
+    - *Mastercard*
+    - *PAYONE Paysafe Pay Later Installment*
+    - *PAYONE Paysafe Pay Later Invoice*
+    - *Sofortüberweisung*
+    - *PAYONE Paysafe Pay Later Debit*
+    - *PAYONE iDeal*
+    - *PAYONE EPS*
+    - *PAYONE Secure Invoice*
+    - *PAYONE Invoice*
+    - *PAYONE Paydirekt*
+    - *PAYONE Bancontact*
+    - *PAYONE Prepayment*
+    - *PAYONE Trustly*
+    - *Direct debit*
+    - *Paydirekt*
```

Comment:
Woher hast du diese Zahlungsmethoden? Aus dem Video? Ich glaube nicht, dass die alle immer verfügbar sind, gerade, weil auch ratepay darunter ist. Ggf. als Frage aufnehmen, ob es vorinstallierte Zahlungsmethoden in Shopware 6 gibt oder ob man sich die immer individuell zusammenklicken kann. Alternativ würde ich einfach sowas schreiben wie: "All payment methods configured in Shopware 6 are displayed".